ctgan/data_sampler.py

```diff
@@ -64,8 +66,7 @@ def is_discrete_column(column_info):
                 if log_frequency:
                     category_freq = np.log(category_freq + 1)
                 category_prob = category_freq / np.sum(category_freq)
-                self._discrete_column_category_prob[current_id, :span_info.dim] = (
-                    category_prob)
+                self._discrete_column_category_prob[current_id, :span_info.dim] = (category_prob)
```

Comment:
The parenthesis can be removed here:
```python
self._discrete_column_category_prob[current_id, :span_info.dim] = category_prob
```
I'm not sure why they existed in the first place.